src/hera/workflows/container.py

```diff
@@ -73,7 +74,7 @@ def _build_container(self) -> _ModelContainer:
     def _build_template(self) -> _ModelTemplate:
         """Builds the generated `Template` representation of the container."""
         return _ModelTemplate(
-            active_deadline_seconds=self.active_deadline_seconds,
```

Comment:
Same here - this should be converted for the user. I noticed we switched from int to str in the global config example, which is a breaking change. It works in Python but users' builds/checks might fail

Reply:
The global config example was an accidental change while fixing the upstream examples. Reverted.